$OpenBSD: patch-configure,v 1.1 2012/09/21 13:51:56 jasper Exp $

Don't pickup pulseaudio if it just happened to be installed.

--- configure.orig	Fri Sep 21 15:33:23 2012
+++ configure	Fri Sep 21 15:33:38 2012
@@ -18054,7 +18054,7 @@ fi
 
 
 
-for ac_header in arpa/inet.h limits.h netinet/in.h  netinet/ip.h netdb.h stdarg.h sys/ioctl.h sys/types.h sys/socket.h syslog.h pulse/pulseaudio.h termios.h
+for ac_header in arpa/inet.h limits.h netinet/in.h  netinet/ip.h netdb.h stdarg.h sys/ioctl.h sys/types.h sys/socket.h syslog.h termios.h
 do
 as_ac_Header=`$as_echo "ac_cv_header_$ac_header" | $as_tr_sh`
 if { as_var=$as_ac_Header; eval "test \"\${$as_var+set}\" = set"; }; then
